include/cliargsparser.h

```diff
@@ -73,7 +73,7 @@ class CliArgsParser {
 	/// If non-null, Newsboat should execute these commands.
 	///
 	/// \note The parser does not check if the passed commands are valid.
-	nonstd::optional<std::vector<std::string>> cmds_to_execute() const;
+	std::vector<std::string> cmds_to_execute() const;
```

Comment:
Please update the docstring as well; "non-null" makes little sense for `std::vector`.